jobs/ostree-compose.yml

```diff
@@ -59,40 +59,30 @@
               which repo of the package being updated
     builders:
         - shell: |
-            cat << EOF > ${WORKSPACE}/job.properties_running
-                topic=org.centos.prod.ci.pipeline.compose.running
-                build_url=$BUILD_URL
-                compose_url=http://artifacts.ci.centos.org/artifacts/fedora-atomic/${fed_branch}/ostree
-                compose_rev=
-                build_id=$BUILD_ID
-                ref=fedora/${fed_branch}/x86_64/atomic-host
-                rev=${rev}
-                namespace=rpms
-                repo=$fed_repo
-                status=success
-                test_guidance=
-            EOF
+            if [ "$fed_branch" = "master" ]; then
+                echo "fed_branch=rawhide" > ${WORKSPACE}/job.properties
+            fi
         - inject:
-            properties-file: ${WORKSPACE}/job.properties_running
+            properties-file: ${WORKSPACE}/job.properties
         - jms-messaging:
             override-topic: org.centos.prod.ci.pipeline.compose
```

Comment:
Yours is not variable.  This is your code:
topic=org.centos.prod.ci.pipeline.compose.running
That is static.  It will never change. So in this publisher, the override-topic can match it.  Mine is not static.  This is mine
topic=${{topic}}
Which I get from injecting.  However, you cannot use ${{topic}} in override-topic just yet, so I cut off my topic in the override-topic line.  Not sure I follow you how yours can change when the topic doesn't have any variables in it, just static text